practice/src/css/components/button.css

```diff
@@ -1,26 +1,56 @@
 .btn {
   cursor: pointer;
   font-family: var(--primary-font);
-  font-weight: var(--font-weight-medium);
 }
 
 .btn-primary {
   border: none;
   border-radius: 12px;
-  background: var(--tertiary-background-color);
   color: var(--primary-text-color);
   font-size: var(--font-size-medium);
-
-  &:hover {
-    background: var(--primary-background-color);
-    color: var(--secondary-text-color);
-  }
+  font-weight: var(--font-weight-medium);
+  line-height: var(--secondary-line-height);
+  padding: 14px 24px;
 }
 
 .btn-secondary {
-  border-radius: 100px;
   border: 1.5px solid var(--primary-border-color);
   background: var(--primary-background-color);
   color: var(--success-text-color);
   font-size: var(--font-size-xxsmall);
-}
\ No newline at end of file
+
+  &:hover {
+    color: var(--primary-text-color);
+    background-color: var(--tertiary-background-color);
+  }
+}
+
+.btn-search-banner {
```

Comment:
If you only use this class for banner section, please move it to best-place.css file